core/templates/pages/topic-viewer-page/topic-story-section/topic-lesson-card/topic-lesson-card.component.css

```
@@ -1,71 +1,211 @@
 }
 
 .topic-lesson-card-description {
   color: #3d3d3d;
   font-family: Roboto, Arial, sans-serif;
-  font-size: 20px;
-  line-height: 1.2;
+  font-size: 17px;
+  line-height: 24px;
   margin: 0;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)